src/screens/ChatInfoScreen/ChatInfoScreen.tsx

```diff
@@ -97,7 +101,7 @@ export const styles = StyleSheet.create({
     fontWeight: 'bold'
   },
   lineTop: {
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The change from `COLORS.lightgrey` to `COLORS.gray100` for the `backgroundColor` property in the `lineTop` style is a minor one. However, ensure that this new color still provides enough contrast against any elements that may be overlaid on it.
```diff
-    backgroundColor: COLORS.lightgrey,
+    backgroundColor: COLORS.gray100,
```



<!-- This is an auto-generated comment by OSS CodeRabbit -->